SECURITY.md

```diff
@@ -0,0 +1,15 @@
+# Security policy
+
+## Supported versions
+
+The versions of the project that are currently supported with security updates.
+
+| Version | Supported          |
+| ------: | :----------------- |
+|     5.x | :white_check_mark: |
+|     4.x | :white_check_mark: |
+|   < 4.0 | :x:                |
```

Comment:
We are also still supporting 3.x (until end of March 2024)

Reply:
fixed